lib/tdf3/src/crypto/declarations.ts

```diff
@@ -21,12 +22,14 @@ export type PemKeyPair = {
  */
 export const MIN_ASYMMETRIC_KEY_SIZE_BITS = 2048;
 
-export type CryptoService = {
+export type CryptoService<PairT = CryptoKeyPair> = {
```

Comment:
what does this mean?

Reply:
This is to allow the types of the keys to vary with the crypto service. Notably, a custom provider may keep its keys in its own database or HSM or something, so it might not want to keep the CryptoKey/CryptoKeyPair interface